cli/status.go

```diff
@@ -156,13 +157,22 @@ type rs struct {
 }
 
 type node struct {
-	Host string   `json:"host"`
-	Ver  string   `json:"agent"`
-	OK   bool     `json:"ok"`
-	Errs []string `json:"errors,omitempty"`
+	Host  string   `json:"host"`
+	Ver   string   `json:"agent"`
+	Arb   *bool    `json:"arbiter,omitempty"`
+	Delay *int64   `json:"delaySecs,omitempty"`
+	OK    bool     `json:"ok"`
+	Errs  []string `json:"errors,omitempty"`
 }
 
 func (n node) String() (s string) {
+	if n.Arb != nil && *n.Arb {
+		return fmt.Sprintf("%s: arbiter node is not supported", n.Host)
```

Comment:
Since we're checking each replset, can we do something like this:
```
Cluster:
========
config:
  - config/localhost:37027 [P]: pbm-agent v2.0.0 OK
  - config/localhost:37028 [S]: pbm-agent v2.0.0 OK
  - config/localhost:37029 [S]: pbm-agent v2.0.0 OK
rs1:
  - rs1/localhost:37018 [P]: pbm-agent v2.0.0 OK
  - rs1/localhost:37019 [!Arbiter]: Not Supported
  - rs1/localhost:37020 [S]: pbm-agent v2.0.0 OK
rs2:
  - rs2/localhost:28018 [P]: pbm-agent v2.0.0 OK
  - rs2/localhost:28019 [S]: pbm-agent v2.0.0 OK
  - rs2/localhost:28020 [!Delayed]: Not Supported
```

Where [P] and [S] are "Primary" and "Secondary" respectively. I intentionally left only P and S not to overload the displayed data. But Arbiter and Delayed are spelt in full to draw attention + it's not so frequent case.